docker-compose.yml

```diff
@@ -26,30 +28,30 @@ services:
     init: true
 
   postgres:
-    # image: "pgautoupgrade/pgautoupgrade:latest"
-    build: ./dockerfiles/postgres
+    image: "pgautoupgrade/pgautoupgrade:latest"
```

Comment:
Could not find `postgres-13.x` (assuming we are using `postgres-13.x` in production) version of `pgautoupgrade`. I would have preferred to use the same version as production for local development.